marlowe-chain-sync/marlowe-chain-sync.cabal

```diff
@@ -107,6 +107,7 @@ library libchainseek
     , these
     , transformers
     , vector
+  visibility: public
```

Comment:
Thanks!

Even with the visibility set, it appears that either `haddock` is not indexing these sub-libraries, or `hoogle` cannot find the sub-library indices. This is a major headache for service API documentation. I couldn't find a quick fix, so I'll create a ticket to investigate and resolve this problem.

Reply:
Hm, shoot...